components/publish/QuickInfo.tsx

```diff
@@ -76,7 +76,7 @@ export const Chip = styled.div`
     background: var(--bs-body-bg);
     display: flex;
     flex-direction: column;
-    padding: 2rem 1rem 1rem 1rem;
+    padding: 1rem 1rem 1rem 1rem;
```

Comment:
nit: If these are all the same now, should we just use the shorthand `padding: 1rem`? Or, come to think of it, use the Bootstrap utility classes (e.g. `p-3`)?

Reply:
done